Gemfile

```diff
@@ -61,3 +61,5 @@ group :test do
   gem 'simplecov', require: false
   gem 'webdrivers', '~> 5.0.0'
 end
+
+gem 'scenic', '~> 1.6'
```

Comment:
Perhaps it is better to put this gem in a group?

Reply:
Put into group of general